### DISCUSSION TITLE
Idiomatic install and package management?

### DISCUSSION THREAD

**Discussion by hyperswine** [FStarLang/FStar#2791] [Q&A]
Is there a goto set of tools for F* like how F# has dotnet? I oculdnt find too much info on this

**Comment by klinvill** [comment]
There's no set of tools for package management for F* yet. There was some discussion about this in #2529, but I'm not aware of any progress beyond those initial discussions.

**Reply by briangmilnes** [reply]
Nik has proposed we put this on the queue for an upcoming meeting and talk me into building some. See #2907 the
makefile that I constructed that builds OCaml output into a dune configuration and runs that. Also there
is https://github.com/FStarLang/FStar/tree/master/examples/hello/multifile which does the verification straight in fstar but does not setup ocaml execution.
